uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/API/Query/LeaveRequestSelect.php

```diff
@@ -86,17 +89,27 @@ private function addWhere(CRM_Utils_SQL_Select $customQuery) {
         )'
     ];
 
+    if($hasUnassignedAsFalse) {
+      $conditions[] = implode(' AND ', $this->hasActiveLeaveManagerCondition());
```

Comment:
Yes because without the `$this->hasActiveLeaveManagerCondition())` returns an array and adding it to the conditions array makes it become an array of arrays.